support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -10,16 +10,18 @@ import {
 	getVariantsAsString,
 	targetPageMatches,
 } from '../abtest';
-import type { Participations } from '../abtest';
+import type { Participations, Tests } from '../abtest';
 
 const { subsShowcaseAndDigiSubPages, digiSub } = pageUrlRegexes.subscriptions;
 const { nonGiftLandingNotAusNotUS, nonGiftLandingAndCheckoutWithGuest } =
 	digiSub;
+
 jest.mock('ophan', () => ({
 	record: () => null,
 }));
+
 // ----- Tests ----- //
-const emptySettings: Settings = {
+const emptySettings = {
```

Comment:
This object doesn't conform to the `Settings` type so removed it here and am type casting the variable when it is called in the tests